tests/lit-tests/1238.ispc

```diff
@@ -0,0 +1,52 @@
+// RUN: %{ispc} --arch=x86-64 --pic -O3 -DINLINE=inline --target=sse4-i32x8,avx1-i32x8,avx2-i32x8 --nostdlib --nowrap -o %t.ll --emit-llvm-text %s 2>&1 | FileCheck %s
```

Comment:
I'd simplify this test. `-DINLINE` can be replaced with explicit `inline` specifier. Also NoisePerm can be reduced in size.

Reply:
I have reduced it.